src/pipeline/world/collision_world.rs

```diff
@@ -381,6 +399,52 @@ impl<N: RealField, T> CollisionWorld<N, T> {
         }
     }
 
+    /// Returns all objects in the collision world that intersect with the shape
+    /// transformed by `isometry` along `direction` until `maximum_distance` is
+    /// reached. The objects are not returned in any particular order. You may
+    /// use the `toi` returned for each object to determine the closest object.
+    #[inline]
+    pub fn sweep_test<'a>(
+        &'a self,
+        shape: &'a Shape<N>,
+        isometry: &'a Isometry<N>,
+        direction: &'a Unit<Vector<N>>,
+        maximum_distance: N,
+        groups: &'a CollisionGroups,
+    ) -> impl Iterator<Item = (CollisionObjectHandle, SweepTestIntersection<N>)> + 'a
+    {
+        let a = shape.local_aabb().transform_by(&isometry);
+        let b = shape.local_aabb().transform_by(&Isometry::from_parts(
+            Translation::from(isometry.translation.vector + direction.as_ref() * maximum_distance),
+            Rotation::identity(),
+        ));
```

Comment:
This should be `let b = shape.aabb(&Isometry::from_parts(...))`.

Reply:
Done!